### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DennisLfromGA**
Add tip about sharing files and folders.

### CODE REVIEW

**Comment by dnschneid** [comment]
Thanks!  I'd prefer to avoid links to the wiki in the README, since it should be mostly standalone.  Could you instead reference the /etc/crouton/shares folder, and (optionally) suggest that more hints are in the wiki?  When in doubt, keep it short.

File: README.md [hunk 1]
```diff
@@ -219,6 +219,8 @@ Tips
     `croutonpowerd -i command and arguments` will automatically stop inhibiting
     power management when the command exits.
   * Have a Pixel or two or 4.352 million? `-t touch` improves touch support.
+  * Want to share some files and/or folders between ChromeOS and your chroot?
+    Then use the `/etc/crouton/shares` file, read all about it in the wiki: Sharing files and folders.
```
Comment (dnschneid) [inline README.md]: Tweaks:

```
Want to share some files and/or folders between Chromium OS and your chroot?
Check out the `/etc/crouton/shares` file, or read all about it in the wiki.
```

Comment (DennisLfromGA) [inline README.md]: Done'ish

On Thu, Jan 22, 2015 at 2:59 PM, David Schneider <notifications@github.com>
wrote:

> In README.md
> <https://github.com/dnschneid/crouton/pull/1369#discussion_r23404847>:
>
> > @@ -219,6 +219,8 @@ Tips
> >      `croutonpowerd -i command and arguments` will automatically stop inhibiting
> >      power management when the command exits.
> >    * Have a Pixel or two or 4.352 million? `-t touch` improves touch support.
> > +  * Want to share some files and/or folders between ChromeOS and your chroot?
> > +    Then use the `/etc/crouton/shares` file, read all about it in the wiki: Sharing files and folders.
>
> Tweaks:
>
> Want to share some files and/or folders between Chromium OS and your chroot?
> Check out the `/etc/crouton/shares` file, or read all about it in the wiki.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dnschneid/crouton/pull/1369/files#r23404847>.
>



-- 
*DennyL@GMail*

**Comment by DennisLfromGA** [comment]
I'm thinking it would it look better if I added a line break before the 'Check out...' ?

**Comment by dnschneid** [comment]
Apparently you can add two spaces at the end of a line to force a line break.  Try it out and see?

**Comment by DennisLfromGA** [comment]
Apparently so, thanx. I've been using <br> which is kind of kludgy...

On Thu, Jan 22, 2015 at 4:57 PM, David Schneider <notifications@github.com>
wrote:

> Apparently you can add two spaces at the end of a line to force a line
> break. Try it out and see?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dnschneid/crouton/pull/1369#issuecomment-71107340>.
>



-- 
*DennyL@GMail*

**Comment by DennisLfromGA** [comment]
I know it's a little late to ask :( but would it be a good idea to add a similar blurp to the [wiki Home page](https://github.com/dnschneid/crouton/wiki#usage-tips--tricks) in addition to  or instead of the README ?

**Comment by dnschneid** [comment]
I think it makes sense to have it in the README.  Feel free to add it to the wiki as well.

**Comment by DennisLfromGA** [comment]
Will do, thanx for the merge :)

( only 4 attempts for 2 lines - I'm gettin' better ;-)

On Thu, Jan 22, 2015 at 5:17 PM, David Schneider <notifications@github.com>
wrote:

> Merged #1369 <https://github.com/dnschneid/crouton/pull/1369>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dnschneid/crouton/pull/1369#event-223350875>.
>



-- 
*DennyL@GMail*

**Comment by DennisLfromGA** [comment]
wiki updated - https://github.com/dnschneid/crouton/wiki#usage-tips--tricks

**Pull request merged**